beacon_node/eth1/src/service.rs

```diff
@@ -461,10 +466,9 @@ impl Config {
 impl Default for Config {
     fn default() -> Self {
         Self {
-            endpoints: vec![SensitiveUrl::parse(DEFAULT_ETH1_ENDPOINT)
-                .expect("The default Eth1 endpoint must always be a valid URL.")],
+            endpoints: Eth1Endpoint::NoAuth(vec![SensitiveUrl::parse(DEFAULT_ETH1_ENDPOINT)
```

Comment:
Also think this should be in the clap defaults

Reply:
The reason this isn't a clap default is so that users can choose not to run an eth1 endpoint if they wish. Pre-merge, non-staking nodes can omit `--eth1-endpoints` and they don't need to run an "eth1 node" at all.